### DISCUSSION TITLE
How can I send drop_pending_updates via SetWebhookParams?

### DISCUSSION THREAD

**Discussion by ptah-alexs** [mymmrac/telego#206] [Q&A] [answered]
How can I send drop_pending_updates via SetWebhookParams?

**Comment by mymmrac** [comment, marked as answer]
Pass `DropPendingUpdates` in `SetWebhookParams` as `true`?
```go
&SetWebhookParams{
    DropPendingUpdates: true,
}
```

Are there any issues with that?

**Reply by ptah-alexs** [reply]
Thank you, it works. First, I tried `drop_pending_updates: true`  option from [telegram doc](https://core.telegram.org/bots/api#getting-updates), but it doesn't work.
